src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -5,6 +5,31 @@ import { getJWTRequest, TokenType, validateQsh } from "~/src/jira/util/jwt";
 import { Installation } from "~/src/models/installation";
 import { moduleUrls } from "~/src/routes/jira/atlassian-connect/jira-atlassian-connect-get";
 import { matchRouteWithPattern } from "~/src/util/match-route-with-pattern";
+import { getJiraClient } from "~/src/jira/client/jira-client";
+
+export const setJiraAdminPrivileges = async (req: Request, claims, jiraHost, installationId) => {
+	// We only need to add this to the session if it doesn't exist
+	if (req.session.isJiraAdmin !== undefined) {
+		return;
+	}
+	try {
+		const userAccountId = claims.sub;
+		const jiraClient = await getJiraClient(
+			jiraHost,
+			installationId,
+			undefined
```

Comment:
undefined for githubappid.... I dont think it matters, this is for us to ask Jira about Jira permissions so the GitHub app should be irrelevant, the previous args are required for the function though